test/script/regression.py

```diff
@@ -366,7 +379,8 @@ def rebuild(test_data):
 
         # Copy files to gold
         try:
-            shutil.copy(dbinpth, dboutpth)
+            if not test_data.isMultiUser:
+                shutil.copy(dbinpth, dboutpth)
```

Comment:
Please add a comment to indicate we only have a database file to copy whenm we are doing as single-user case test.